submissions/opeyemiPeter/database/init.sql

```diff
@@ -0,0 +1,176 @@
+  ( 3, 4,'CHUNKY SOLE SNEAKERS', 'Sequi laboriosam consequatur possimus id Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias animi sequi asperiores quos, nulla eaque, in, alias earum optio tenetur repellat provident at.', load_file('../html/Images/l.png'), ' 1200.00', 30,'available'  );
+
+-- view product table;
+select * from product;
+
+-- 4 customer table 
+-- Table creation
+create table customers(
+  customerId int not null auto_increment,
+  customerFirstName varchar(50) not null,
+  customerLastName varchar(50) not null,
+  customerEmailAddress varchar(50) not null,
+  customerPassword varchar(50) not null,
+    primary key(customerId)
+);
+
+-- populating the customers table with values
+insert into customers(customerFirstName, customerLastName, customerEmailAddress, customerPassword) 
+values 
+ ('fred', 'hammod', 'fredhamod@gmail.com','frednard'),
+ ('shnano', 'craig', 'fcraig@gmail.com' ,'shanaonocra'),
+ ('fiyinfolu', 'clement', 'clement@gmail.com', 'clementfiyin'),
+ ('suite', 'funmi', 'funmisuite@gmail.com', 'suiteees');
+
+-- view customers page 
+select * from Customers;
+
+
+-- 5. Address Table 
+create table CustomersAddress(
+ addressId int not null auto_increment,
+ customerId int not null,
+ customerStreet varchar(200) not null,
+ customerState varchar(30) not null,
+ customerCity varchar(30) not null,
+ customerZipCode mediumint,
+ customerCountry varchar(20) not null,
+ customerPhone varchar(15) not null,
+ primary key (addressId),
+ foreign key(customerId) references Customers(customerId)
+);
+drop table CustomersAddress;
+
+-- populating the customeraddress table
+insert into CustomersAddress(customerId, customerStreet, customerState, customerCity, customerZipCode, customerCountry, customerPhone )values 
+ (1, ' no 2 New Bodija', 'Oyo', 'Ibadan', 23454, 'Nigeria','+234567890'),
+ (2, ' no 45 New Garage', 'Oyo', 'Ibadan', 23454, 'Nigeria','+2309876543'),
+ (3, ' no 34 Moniya', 'Oyo', 'Ibadan', 23454, 'Nigeria','+234564321'),
+ (4, ' no 478 Iwo road', 'Oyo', 'Ibadan', 23454, 'Nigeria','+2345654456');
+
+-- view customers address
+select * from CustomersAddress;
+
+-- 6 Order table
+-- table creation 
+create table Orders(
+ orderId int not null auto_increment,
+ clientOrder int not null,
+ orderAmount decimal(25,2) not null,
+ orderCreateAt datetime,
+ primary key  (orderId),
+ foreign key (clientOrder) REFERENCES Customers (customerId)
+);
+-- populationg the orders with values
+
+insert into Orders (clientOrder,orderAmount,orderCreateAt)
+values 
+ (1 , 1000.00, now()),
+ (2, 3000.00, now()),
+ (3, 3000.00, now()),
+ (4, 1200.00, now());
+
+-- view Order table
+
+select * from Orders;
+
+-- 7. Order Item
+-- creation of the Table
+create table orderItem(
+ orderItemId int not null auto_increment,
+ OrderId int not null,
+ productId int not null,
+ quantity int not null,
+ unitPrice int not null,
```

Comment:
kindly use `decimal` for anything relating to amount

Reply:
@ i have changed it.

Reply:
@adaugochi

Reply:
@adaugochi I have used decimal as directed. thanks